.github/workflows/s3-deploy.yml

```diff
@@ -0,0 +1,54 @@
+name: Deploy to S3
+
+on:
+  push:
+    branches: [ dev ]
+  pull_request:
+    branches: [ dev ]
+  release:
+    types: [ published, created, edited ]
```

Comment:
Probably only want published here, otherwise this will upload anytime we edit the description of the release.